zeppelin-jupyter-interpreter/src/main/java/org/apache/zeppelin/jupyter/JupyterKernelInterpreter.java

```diff
@@ -216,6 +259,23 @@ private void launchJupyterKernel(int kernelPort)
     }
   }
 
+  private File buildBootstrapScriptFile(int kernelPort) throws IOException {
+    StringBuilder builder = new StringBuilder();
+    if (condaEnv != null) {
+      builder.append("source " + condaEnv + "/bin/activate\n");
+    }
+    builder.append(pythonExecutable);
+    builder.append(" " + kernelWorkDir.getAbsolutePath() + "/kernel_server.py");
```

Comment:
If you use the StringBuilder, you should use the String Builder for the entire string.
```
builder.append(" ").append(kernelWorkDir.getAbsolutePath()).append("/kernel_server.py");
...
```